doc/api/async_hooks.md

```diff
@@ -973,6 +973,21 @@ chains. It allows storing data throughout the lifetime of a web request
 or any other asynchronous duration. It is similar to thread-local storage
 in other languages.
 
+## Difference with raw Async Hooks
+
+`AsyncLocalStorage` doesn't provide any new functionality that raw Async Hooks
+doesn't already provide: anything implemented with `AsyncLocalStorage` can be
+implemented with Async Hooks instead.
+
+The goal of `AsyncLocalStorage` is to provide a more performant and memory safe
```

Comment:
This paragraph only seems to claim that ALS is faster (than what?) and describe implementation details (which might change in future, so I'm not sure if that's a good idea to mention them). Does it add any value to what we have now?